### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by tijmenb**
This updates the readme to reflect the current state of the app.

For review, look at the complete document, not just the changes:

https://github.com/alphagov/collections-publisher/blob/5fa8f74ae22d089d5c3743a7d6ce1b3599460134/README.md

Trello: https://trello.com/c/d3RlTJwt

I'd like this reviewed at least two people on @alphagov/team-finding-things.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,36 @@
...
+- [collections-publisher](https://collections-publisher.publishing.service.gov.uk/)
+
 ## Nomenclature
 
 See the [README of collections frontend](https://github.com/alphagov/collections).
 
 ## Technical documentation
 
-This is a Ruby on Rails application for internal use, with no public facing aspect. It retrieves information about which content is tagged to a topic or browse page from Rummager and publishes Curated lists to the Content store.
+This is a Ruby on Rails application for internal use by GDS and some departemental editors. It retrieves information about which content is tagged to a topic or browse page from [rummager](https://github.com/alphagov/rummager) and publishes curated lists and taxons to the [publishing-api](https://github.com/alphagov/publishing-api).
```
Comment (MatMoore) [inline README.md]: s/departemental/departmental

**Comment by MatMoore** [comment]
Other than a small typo, looks good to me. Live examples for browse and topic pages are really useful. :+1:

**Comment by mobaig** [comment]
Are you using the term 'curated lists' to collectively refer to both content tagged to topics and content tagged to browse pages? If so, I wonder if you can replace this with something more explicit - 'curated lists' sounds like you might be introducing a new concept. Potentially a bit confusing.

**Comment by Davidslv** [comment]
Looks good, I agree with @mobaig on the `curated lists`

**Comment by tijmenb** [comment]
@mobaig @Davidslv well spotted - I _was_ actually introducing a new concept. I've updated the PR to include more about these "curated lists".

**Comment by MatMoore** [comment]
:ship:

**Pull request merged**